[2022-11-29 09:51:59,062] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: demo.demo2 scheduled__2022-11-28T00:00:00+00:00 [None]>
[2022-11-29 09:51:59,066] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: demo.demo2 scheduled__2022-11-28T00:00:00+00:00 [None]>
[2022-11-29 09:51:59,066] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-11-29 09:51:59,066] {taskinstance.py:1239} INFO - Starting attempt 1 of 1
[2022-11-29 09:51:59,066] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-11-29 09:51:59,072] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): demo2> on 2022-11-28 00:00:00+00:00
[2022-11-29 09:51:59,074] {standard_task_runner.py:52} INFO - Started process 65716 to run task
[2022-11-29 09:51:59,078] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'demo', 'demo2', 'scheduled__2022-11-28T00:00:00+00:00', '--job-id', '7', '--raw', '--subdir', 'DAGS_FOLDER/demo.py', '--cfg-path', '/var/folders/jd/dj9qbrrx6jdc6f5lwwg0h7tm5np56x/T/tmp3g94e4jg', '--error-file', '/var/folders/jd/dj9qbrrx6jdc6f5lwwg0h7tm5np56x/T/tmpo00pzla6']
[2022-11-29 09:51:59,079] {standard_task_runner.py:77} INFO - Job 7: Subtask demo2
[2022-11-29 09:52:29,116] {logging_mixin.py:109} INFO - Running <TaskInstance: demo.demo2 scheduled__2022-11-28T00:00:00+00:00 [running]> on host LM0002893
[2022-11-29 09:52:59,154] {taskinstance.py:1424} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=demo
AIRFLOW_CTX_TASK_ID=demo2
AIRFLOW_CTX_EXECUTION_DATE=2022-11-28T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-11-28T00:00:00+00:00
[2022-11-29 09:52:59,155] {logging_mixin.py:109} INFO - second task
[2022-11-29 09:52:59,155] {python.py:175} INFO - Done. Returned value was: None
[2022-11-29 09:52:59,160] {taskinstance.py:1267} INFO - Marking task as SUCCESS. dag_id=demo, task_id=demo2, execution_date=20221128T000000, start_date=20221129T042159, end_date=20221129T042259
[2022-11-29 09:53:09,148] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-11-29 09:53:09,161] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
